docs/guide/en/pure-fields.md

```diff
@@ -1,3 +1,90 @@
 # Pure fields
 
-PureField helper and PureFieldFactory (don't forget about inheritance)
+To ease the creation of the fields with `\Yiisoft\Form\Field\Base\InputData\PureInputData` as a data source, use
+corresponding helper (`\Yiisoft\Form\PureField`):
+
+```php
+use Yiisoft\Form\PureField;
+
+$field = PureField::text('name', 'value');
```

Comment:
@vjik is it possible to just call these Pure* classes, such as PureField, simpler? For example, PureField → Field, PureInput → Input?

Reply:
#328